modules/lib/src/main/resources/assets/js/page-editor/LiveEditPage.ts

```diff
@@ -515,66 +518,73 @@ export class LiveEditPage {
         UpdateTextComponentViewEvent.un(this.updateTextComponentViewListener);
     }
 
-    public loadComponent(componentView: ComponentView, componentUrl: string,): Q.Promise<string> {
-        const deferred = Q.defer<string>();
+    public loadComponent(componentView: ComponentView, componentUrl: string): Promise<void> {
         assertNotNull(componentView, 'componentView cannot be null');
         assertNotNull(componentUrl, 'componentUrl cannot be null');
 
         componentView.showLoadingSpinner();
 
-        $.ajax({
-            url: componentUrl,
-            type: 'GET',
-            success: (htmlAsString: string) => {
-                const newElement: Element = this.wrapLoadedComponentHtml(htmlAsString, componentView.getType());
-                const itemViewIdProducer: ItemViewIdProducer = componentView.getItemViewIdProducer();
-                const itemViewFactory: ItemViewFactory = componentView.getItemViewFactory();
-
-                const createViewConfig: CreateItemViewConfig<RegionView> = new CreateItemViewConfig<RegionView>()
-                    .setItemViewIdProducer(itemViewIdProducer)
-                    .setItemViewFactory(itemViewFactory)
-                    .setLiveEditParams(this.pageView.getLiveEditParams())
-                    .setParentView(componentView.getParentItemView())
-                    .setPositionIndex(componentView.getPath().getPath() as number)
-                    .setElement(newElement);
-
-                const newComponentView: ComponentView = itemViewFactory.createView(
-                    componentView.getType(),
-                    createViewConfig) as ComponentView;
-
-                componentView.replaceWith(newComponentView);
-                const parentItemView = newComponentView.getParentItemView();
-
-                if (parentItemView instanceof RegionView) { // PageView for a fragment|
-                    newComponentView.getParentItemView().registerComponentViewListeners(newComponentView);
+        return fetch(componentUrl)
+            .then(response => {
+                const hasContributions = response.headers.has('X-Has-Contributions');
+
+                // reloading entire live page if the component has contributions and there are no equal components on the page
+                if (hasContributions && !this.hasSameComponentOnPage(componentView.getPath())) {
+                    NotifyManager.get().showFeedback(i18n('live.view.component.has.contributions'));
```

Comment:
We don't need to show the notification. Editors should not/will not care about page contributions and it's too much text (in addition to notification about saved changes) which they will not have time to read. Remove the new i18n key as well.